app.js

```diff
@@ -6,4 +67,80 @@ function toggleMenu() {
   menuActive.classList.toggle('active');
   logoActive.classList.toggle('active');
 }
-module.exports = toggleMenu();
\ No newline at end of file
+
+const createPopUp = document.querySelector('.csubmit');
+createPopUp.addEventListener('click', () => {
+  const pop = document.querySelector('body');
+  const section = document.createElement('section');
+  section.className = 'pop-up-section';
+  pop.appendChild(section);
+  section.innerHTML = `
+  <div class="pop-up">
+  <div class="pop-up-box1">
+    <div class="pop-up-box-title">
+      <h1 class="pop-up-title"></h1>
+      <div class="pop-up-x"></div>
+    </div>        
+    <ul class="pop-up-ul">
+      <li class="pop-up-li" id="lenguage1"></li>
+      <li class="pop-up-li" id="lenguage2"></li>
+      <li class="pop-up-li" id="lenguage3"></li>
+    </ul>
+  </div>
+  <div class="pop-up-box2">
+      <div class="pop-up-arrow" id="arrow1"></div>
+    <img class="pop-up-image" src="">
+    <div class="pop-up-secundary-box1">
+      <p class="pop-up-p">
+      </p>
+      <div class="pop-up-secundsry-box2">
+        <a class="pop-up-button" href="https://nimplay.github.io/Microverse_Portfolio/#">
+          <p>See Live</p>
+          <img src="./Images/liveServer.svg" alt="">
+        </a>
+        <a class="pop-up-button" href="https://github.com/nimplay/Microverse_Portfolio">
+          <p>See Source</p>
+          <img src="./Images/whiteGitHub.svg" alt="">
+        </a>              
+      </div>
+    </div>
```

Comment:
- [ ] I can see that you are manually displaying only one card. The idea here is that for the array of projects you already created, you need to loop through that array and render the lopped data in the section where you're manually displaying a single project. That way, all six projects will get displayed on the screen just as in the Figma template. 
- [ ] For this project you are not required to make the project's modal displayed in a carousel fashion. You are required to have six projects displayed on the screen and when someone clicks on a single project to view its details, the details for that project are then displayed on the popup window. Please use that method in order for your project to be completed. I trust you can 👍 
- [ ] Also, please work on the image that is overlapping the badges on your card or cards(when you must have implemented the lopped cards). Please work on your CSS styles to correctly align card items on your project.

**See your Project**
![image](https://user-images.githubusercontent.com/50941074/173930756-5da15a86-61ef-4023-958a-4c494e486e52.png)

**The Figma Template**
![image](https://user-images.githubusercontent.com/50941074/173931115-1353e17d-72f2-4cc2-96b7-985e2fbe3c95.png)